CMSgov/mint-app
# [EASI-3693] Update key characteristics question

# EASI-3693

## Changes and Description

- add two new options
- update one option



## How to test this change

1. Page 2 of General characteristics
2. dropdown to match figma
4. Check data persists in csv, read only, and reload



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
